code/datums/gamemode/role/cultist.dm

```diff
@@ -28,10 +28,29 @@
 	var/datum/role/cultist/mentor = null
 	var/list/acolytes = list()
 
+	var/devotion = 0
+	var/rank = DEVOTION_TIER_0
```

Comment:
Better to make this a global/const/whatever global associative list, i.e.
{DEVOTION_TIER_1, 100}, {DEVOTION_TIER_2, 500}, {DEVOTION_TIER_3, 1000}
And then edit the places in this code to look at that so that editing that list will edit everything else.